Resources/public/js/views/ez-fieldeditview.js

```diff
@@ -229,7 +216,26 @@ YUI.add('ez-fieldeditview', function (Y) {
          * @method initializer
          */
         initializer: function () {
-            Y.eZ.TemplateBasedView.registerPartial('ez_fieldinfo_tooltip', 'fieldinfo-tooltip-ez-partial');
+            /**
+             * Set if the fieldDefinition description is active or not.
+             *
+             * @property _toggleFieldDefinitionDescription
+             * @protected
+             * @type string
+             * @default 'active'
+             */
+            this._toggleFieldDefinitionDescription = 'active';
+
+            /**
+             * Set if the fieldDefinition description has the standard display
+             *
+             * @property _setDescriptionStandardDisplay
+             * @protected
+             * @type string
+             * @default 'active'
+             */
+            this._setDescriptionStandardDisplay = 'active';
```

Comment:
same here, this should be a boolean property and be called something `_useStandardFieldDefinitionDescription`. You should also explain the documentation of the what does *standard* means in this context.